src/app/locationForm/common/Form.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
```

Comment:
I updated the login in `isEditing`, and generalized `Form.value` so that it can take an object of multiple fields. Please see my solution in https://github.com/streetlives/streetlives-web/pull/8/commits/8723b4ce0c82ccd767c5542d324e568d3635d336#diff-71061816a970b190e63f516087e7325dR18. This is for the Phone Number form, which takes a phone number and extension. Furthermore, it splits up the phone number into three different fields for area code, first three digits, and last four digits.

You can see how these fields get passed back to dispatch in the Edit form here:
https://github.com/streetlives/streetlives-web/pull/8/commits/8723b4ce0c82ccd767c5542d324e568d3635d336#diff-c57e579bad4f4f74984ee84e76c54e7dR55

And dispatched to the reducer here: 
https://github.com/streetlives/streetlives-web/pull/8/commits/8723b4ce0c82ccd767c5542d324e568d3635d336#diff-c57e579bad4f4f74984ee84e76c54e7dR141

Please let me know what you think. Thanks.

Reply:
Yeah, I think this solution should work.
I left a comment on the Phone comment, mentioning I still think it might be more flexible to have the specific forms pass an explicit prop (so Form doesn't have to care about the values at all), but functionally either approach should work.